.github/workflows/interchange_ci.yml

```diff
@@ -0,0 +1,27 @@
+name: FPGA interchange CI tests
+
+on: [push, pull_request]
+
+jobs:
+
+  Run-tests:
+    runs-on: ubuntu-latest
+    steps:
+
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+
+    - uses: actions/setup-python@v2
+
+    - name: Install
+      run: |
+        sudo apt-get update
+        sudo apt-get install git make cmake libboost-all-dev python3-dev libeigen3-dev tcl-dev clang bison flex swig
+
+    - name: Execute build script
+      run: stdbuf -i0 -o0 -e0 ./.github/ci/build_interchange.sh
+
+    - name: Execute test script
+      run: |
+        cd build && make all-fpga_interchange-tests -j`nproc`
```

Comment:
Can you run the equivalent of `make -C create_bba test` in the new system?

Reply:
Sorry, I mean `make -C archcheck check`

Reply:
Sure, I have added an `all-${family}-archcheck-tests` target to accumulate all the archcheck tests dependencies.